base_iso3166/__manifest__.py

```diff
@@ -1,16 +1,16 @@
-# -*- coding: utf-8 -*-
-# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
-# © 2017  Creu Blanca <www.creublanca.es>
+# Copyright 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# Copyright 2017  Creu Blanca <www.creublanca.es>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "ISO 3166",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "author": "Tecnativa, "
               "Creu Blanca, "
+              "AdaptiveCity, "
```

Comment:
I think not significant changes for being co-author.

Reply:
But I added some tests :(

Reply:
@emagdalenaC2i does AdaptiveCity appears somewhere else? I think I did not contribute in the project repository.

Reply:
Well, Aitor, you know that the effort for doing that little test has not been very high. But don't worry, you will soon appear in several modules with the pending refactorings we have in mind and don't forget that you always appear in contributors section.